janusgraph-codepipelines-ci/src/test/java/org/janusgraph/codepipelines/TestContainerOne.java

```diff
@@ -0,0 +1,29 @@
+package org.janusgraph.codepipelines;
+
+import org.junit.Test;
+
+/**
+ * Created by amcp on 2017/02/24.
+ */
+public class TestContainerOne {
+    public static class TraversalAlpha {
+        @Test
+        public void testOne() {
+            //noop
+        }
+        @Test
+        public void testTwo() {
+            //noop
+        }
+    }
+    public static class TraversalBeta {
+        @Test
+        public void testOne() {
+            //noop
+        }
+        @Test
+        public void testTwo() {
+            //noop
+        }
+    }
+}
```

Comment:
They are sample classes for me to run and make sure that I was able to select using command line arguments in the JSON files.